tensorflow/python/ops/collective_ops_test.py

```diff
@@ -346,6 +347,32 @@ def testCollectiveGatherShapeMismatchAcrossDevices(self):
                                    'Shape mismatch'):
         sess.run([c0, c1], options=run_options)
 
+  @test_util.run_deprecated_v1
+  def testCollectiveGatherPolymorphicShape(self):
+    t0 = [0, 1, 2, 3, 4, 5, 6, 7]
+    t1 = [10, 11, 12, 13, 14, 15, 16, 17]
+    t01 = [0, 1, 2, 3, 4, 5, 6, 7, 10, 11, 12, 13, 14, 15, 16, 17]
```

Comment:
nit: can you rename these variables to be more descriptive?  Perhaps `input0, input1, expected_output0, expected_output1`.

Reply:
done